assemble.py

```diff
@@ -366,8 +366,13 @@ def get_args():
                 logger.info('Preloading Restorable Apps...')
                 copy_restorable_apps_to_wic(target, image, fetched_apps[target.name][0], preload_progress)
 
-            logger.info('Preloading Compose Apps...')
-            copy_compose_apps_to_wic(target, args.fetch_dir + "/compose", image, args.token, args.app_shortlist, preload_progress)
+            if target.name not in fetched_apps or target.lmp_version < 87:
```

Comment:
am i missing something subtle about why you don't use `release_info.lmp_version` here?

Reply:
There is no `release_info` in this scope/loop.